CHANGELOG.md

```diff
@@ -12,6 +12,16 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 	https://changelog.md/
 -->
 
+## v1.6.0 (WIP)
+
+- Changed function calls and type names to match the regenerated rest clients
+  using: (#91)
+
+  - `wharf-api` v5.0.0, previously v4.1.0.
+  - `wharf-provider-azuredevops` v2.0.1, previously v1.2.0-rc.
+  - `wharf-provider-github` v2.0.0, previously v2.0.0-rc.
+  - `wharf-provider-gitlab` v1.3.0, previously v1.2.0-rc.
```

Comment:
nit: Stay consistent with other version-changing conventions (<https://iver-wharf.github.io/#/development/changelogs/changelog-conventions?id=dependencies>). We don't have many version changes in this repo, but at least in our other repos we've used "from X to Y" in the CHANGELOG.md.

```suggestion
  - `wharf-api` from v4.1.0 to v5.0.0.
  - `wharf-provider-azuredevops` from v1.2.0-rc to v2.0.1.
  - `wharf-provider-github` from v2.0.0-rc to v2.0.0.
  - `wharf-provider-gitlab` from v1.2.0-rc to v1.3.0.
```